Photon-Tinker/Mesh/Common/MeshSetupFlowUIBase.swift

```diff
@@ -778,7 +778,8 @@ class MeshSetupUIBase : UIViewController, Storyboardable, MeshSetupFlowRunnerDel
     internal func networkPasswordViewCompleted(password: String) {
         flowRunner.setSelectedNetworkPassword(password) { error in
             if error == nil {
-                self.showJoiningNetworkView()
+                //self.showJoiningNetworkView()
```

Comment:
er... based on what?  You want to finish this comment? ;)

Also, you should probably remove that commented-out method, too.

Reply:
`//this will trigger based on JoiningNetworkStarted event` 

This was an accidental early-commit. Since it was pushed, i couldn't ammend. I commited the proper comment with next commit :)